client/src/components/Home/index.js

```diff
@@ -247,11 +247,11 @@ export default function Home(props) {
               <br />
 
               <p>
-                <a href="#" className="header-link">
+                <a href="https://docs.rocket.chat/guides/user-guides" className="header-link">
                   {t("unsigned-home-demo.user-guides")}
-                </a>{" "}
```

Comment:
This causes a minor spacing issue here. I'll cover it up in the next one. We can go ahead!